support/vim/sv.vim

```diff
@@ -0,0 +1,56 @@
+" Vim syntax file
+" Language: Silver
+
+" exit if syntax already loaded
+if exists("b:current_syntax")
+	finish
+endif
+
+" keywords and regexes copied from gedit and google-code-prettyify
+
+syn keyword externals grammar import imports exports build hiding as only option
+
+syn keyword declarations parser parse attribute annotation function local closed nonterminal type occurs on production terminal marking foreign layout disambiguate action global productions lexer class default
+
+syn keyword primitiveTypes IO Boolean Decorated Float Function Integer Production String
+
+syn keyword storageClass abstract aspect concrete inherited synthesized autocopy ignore
+
+syn keyword scopeDeclarations left association right precedence operator dominates submits classes
+
+syn keyword flowOther forwarding forwards to return pluck print
+
+syn keyword flow case of let in end decorate with prefix else forward if new then toString toInt toFloat length
+
+syn keyword ide temp_imp_ide_dcl temp_imp_ide_font font color italic bold
+
+syn keyword kw builder postbuilder exporter folder property string required string wizard new file stub generator product name version option resource
+
+syn keyword boolean false true
+
+syn match comment "\(--.*\)\|\({-\([^-]\|-\+[^-}]\)*-}\)$"
```

Comment:
I think Proper Vim Style is to kick off all regexes with \v to avoid problems with different magic modes.